npm/server.js

```diff
@@ -1,5 +1,6 @@
 const net = require('net'),
-    http = require('http');
+    http = require('http'),
+    enableServerDestroy = require('server-destroy');
```

Comment:
Avoid mixing unrelated changes in future pull requests.

Reply:
Yes, existing tests are failing because of this.